nifi-commons/nifi-record/src/test/java/org/apache/nifi/serialization/record/ResultSetRecordSetTest.java

```diff
@@ -241,6 +243,45 @@ public void testCreateRecord() throws SQLException {
         assertEquals(bigDecimal5Value, record.getValue(COLUMN_NAME_BIG_DECIMAL_5));
     }
 
+    @Test(expected = SQLException.class)
+    public void testCreateSchemaArrayThrowsException() throws SQLException {
+        // given
```

Comment:
Although various tests use different approaches, I recommend removing the given/when/then comments.